content/en/synthetics/guide/or-logic-api-tests-assertions.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Perform OR logic in API tests assertions
+kind: guide
+further_reading:
+    - link: '/synthetics/api_tests'
+      tag: 'Documentation'
+      text: 'Create an API Test'
+    - link: '/synthetics/multistep'
+      tag: 'Documentation'
+      text: 'Create a Multistep API Test'
+    - link: "/getting_started/synthetics/api_test"
+      tag: "Documentation"
+      text: "Get started with HTTP tests"
+---
+
+You want to perform `OR` logic on your [API tests][1] assertions to define several different expected values for a same assertion type. For example, you would like your [HTTP test][2] `status code` assertion to succeed in case your server responds with a `200` **or** a `302`.
```

Comment:
```suggestion
In some cases, you might want to perform `OR` logic on your [API tests][1] assertions in order to define several different expected values for a same assertion type. For example, you would like your [HTTP test][2] `status code` assertion to succeed in case your server responds with a `200` **or** with a `302`.
```